src/components/RestaurantList/RestaurantList.js

```diff
@@ -0,0 +1,60 @@
+import { ThemeProvider } from "@mui/material/styles";
+import {theme} from '../../contexts/MainPalette' 
+import SearchBar from "../Searchbar/SearchBar";
+import  Stack  from "@mui/material/Stack";
+import Card from "../Card/Card";
+import React, { useState, useEffect } from 'react';
+import Header from "../Header/Header";
+// import { useLocation } from "react-router-dom";
+
+function RestaurantList({ title }) {
+    const [changeValue, setChangeValue] = useState('');
+    // const { locationState } = useLocation();
+
+
+   const handleCallback = (data) => {
```

Comment:
Attention au nommage de tes fonctions et de tes variables.

Reply:
RestaurantName and handleSearchBarChange